spec/factories/supplier.rb

```diff
@@ -6,6 +6,7 @@
     name { Faker::Company.name.truncate(30) }
     phone { Faker::PhoneNumber.phone_number }
     address { Faker::Address.street_address }
+    iban { Faker::Bank.iban }
```

Comment:
Is iban required for a supplier? If not, then please put it in a trait if you need it. From [FactoryBot docs](https://www.rubydoc.info/gems/factory_bot/file/GETTING_STARTED.md#Defining_factories):
> If you're creating ActiveRecord objects, that means that you should only provide attributes that are required through validations and that do not have defaults. Other factories can be created through inheritance to cover common scenarios for each class.